tests/lib/nested.sh

```diff
@@ -211,6 +214,9 @@ nested_get_ubuntu_image_url_for_vm() {
         ubuntu-20.10-64*)
             echo "https://cloud-images.ubuntu.com/groovy/current/groovy-server-cloudimg-amd64.img"
             ;;
+        ubuntu-21.04-64*)
+            echo "https://cloud-images.ubuntu.com/hirsute/current/hirsute-server-cloudimg-amd64.img"
```

Comment:
21.04 needs to be enabled in nested suites for this to have an effect right?

Reply:
I already deployed the image to the gce bucket, with that it should be enough. It is because we source this script and it uses the current system independently of the suite being executed.

Reply:
@stolowski, I still I see preseed tests failing with this error, any idea about how to fix it?

2020-12-12 13:18:09 (108 MB/s) - ‘cloudimg.img’ saved [563871744/563871744]

+ mkdir -p /mnt/cloudimg
+ . /home/gopath/src/github.com/snapcore/snapd/tests/lib/preseed.sh
++ pwd
+ mount_ubuntu_image /home/gopath/src/github.com/snapcore/snapd/tests/main/preseed/cloudimg.img /mnt/cloudimg
+ local CLOUD_IMAGE=/home/gopath/src/github.com/snapcore/snapd/tests/main/preseed/cloudimg.img
+ local IMAGE_MOUNTPOINT=/mnt/cloudimg
+ grep nbd
+ lsmod
+ modprobe nbd
++ command -v qemu-nbd
+ systemd-run --system --service-type=forking --unit=qemu-nbd-preseed.service '' --fork -c /dev/nbd0 /home/gopath/src/github.com/snapcore/snapd/tests/main/preseed/cloudimg.img
Failed to find executable : No such file or directory

Reply:
No, I need to investigate to see what changed on 21.04. By the way, I think we should add 'Run nested' to this PR (and close/re-open)?